unit_testing_framework/calculator/calculator.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * Calculator for adding and multiplicating numbers
+ * @class Calculator
+ */
+class Calculator {
+    /**
+        * Creates an instance of Calculator.
+        * @memberof Calculator
+        */
+    constructor() {
+    }
+
+    /**
+       *
+       * Sum all parameters of type Number and returns answer
+       * @param {...Number} values
+       * @return {Number} Sum of numbers
+       * @memberof Calculator
+       */
+    add(...values) {
+        if (Array.isArray(values) && (values.length !== 1) &&
```

Comment:
Why do you use it ```Array.isArray(values)``` ?

Reply:
It was to understand if is it) But it's really unnecessary